### PR TITLE
Change memory requirement

### PR DESCRIPTION
**Pull request by dinithiW**
Add js function to handle memory requirements

### CODE REVIEW

File: wdl2cwl/main.py [hunk 1]
```diff
@@ -33,9 +33,25 @@ def get_ram_min(ram_min: str) -> int:
...
+
+
+def get_ram_min_js(ram_min: str) -> str:
+    """Get memory requirement for user input."""
+    js_str = (
+        '${var unit = inputs["'
+        + ram_min
+        + '"].match(/[a-zA-Z]+/g).join("");\nvar value = parseInt(inputs["'
+        + ram_min
+        + '"].match(/[0-9]+/g));\nvar memory = "";\nif(unit==="GiB"){\n\tmemory = value*1024;\n}\nreturn memory;\n}'
```
Comment (mr-c) [inline wdl2cwl/main.py:52]: Great start! Now to implement the rest of the units: https://github.com/openwdl/wdl/blob/main/versions/1.1/SPEC.md#units-of-storage

**Review by mr-c — CHANGES_REQUESTED** [review]
Great start! Now to implement the rest of the units: https://github.com/openwdl/wdl/blob/main/versions/1.1/SPEC.md#units-of-storage

**Comment by dinithiW** [comment]
Add the rest of storage units for JS expressions

**Comment by dinithiW** [comment]
Is it needed to round the number up in case the answer is a decimal number?

**Comment by mr-c** [comment]
> Is it needed to round the number up in case the answer is a decimal number?

No rounding needed, `float`s are okay: https://www.commonwl.org/v1.2/CommandLineTool.html#ResourceRequirement

**Review by mr-c — APPROVED** [review]

**Pull request merged**